torch/csrc/jit/mobile/flatbuffer_loader.cpp

```diff
@@ -543,22 +556,36 @@ TypePtr FlatbufferLoader::getOrCreateTypeAnnotations(
 mobile::Module parse_and_initialize_mobile_module(
     std::shared_ptr<char> data,
     size_t,
-    c10::optional<at::Device>) {
+    c10::optional<at::Device> device,
+    ExtraFilesMap& extra_files) {
   auto* flatbuffer_module = mobile::serialization::GetMutableModule(data.get());
-  mobile::Module m = FlatbufferLoader().parseModule(flatbuffer_module);
+  mobile::Module m =
+      FlatbufferLoader().parseModule(flatbuffer_module, extra_files);
   m.set_delete_memory(std::move(data));
   return m;
 }
 
 mobile::Module initialize_mobile_module(
     mobile::serialization::Module* flatbuffer_module,
-    c10::optional<at::Device>) {
-  mobile::Module m = FlatbufferLoader().parseModule(flatbuffer_module);
+    ExtraFilesMap& extra_files,
```

Comment:
let's leave those function unchanged. Let user call a separate function for those who need extra files.